models/users.js

```diff
@@ -215,6 +215,39 @@ const fetchUserImage = async (users) => {
   return images;
 };
 
+/**
+ * Add default status
+ * @returns {Promise<string[]>}
```

Comment:
Please update this based on the return value

Reply:
done